adr/0017-zarf-bundle.md

```diff
@@ -0,0 +1,93 @@
+# 17. Bundles
+
+Date: 2023-06-13
+
+## Status
+
+Pending
+
+## Context
+
+Orchestrating Zarf packages into meta-packages is a current weakpoint of Zarf. The core of Zarf was oriented around components as capabilities, but as Zarf packages have scaled, it has been discovered that entire packages are now rapidly becoming modularized capabilities.
+
+Currently there is no official way to enable the deployment, publishing, pulling, and creation of these Zarf packages. Due to this shortfalling, it has led to the community developing such antipatterns as:
```

Comment:
```suggestion
Orchestrating capabilities from multiple Zarf packages into meta-packages is a current weakpoint for Zarf. The core of Zarf was built around components as capabilities, but as Zarf packages have scaled, there has been a need to create a new boundary to manage these capabilities efficiently.

Currently there is no official way to enable the deployment, publishing, pulling, and creation of multiple Zarf packages together, and due to this some in the community have resorted to patterns such as:
```
A bit better wording on some of the why (efficiency) and also wanted to stray away from the "antipattern" wording since it isn't really the communities' fault that they are doing that - to me "antipattern" is something that is well communicated as something to not do which up until recently the Zarf team has not done.